lib/plugins/paragon-webpack-plugin/utils.js

```diff
@@ -0,0 +1,373 @@
+const { sources } = require('webpack');
+const parse5 = require('parse5');
+
+function getDescendantByTag(node, tag) {
```

Comment:
I also suggest adding small documentation for each function in JSDocs format

Reply:
Done :)